lib/pure/asyncftpclient.nim

```diff
@@ -360,26 +415,26 @@ proc doUpload(ftp: AsyncFtpClient, file: File,
   assert ftp.dsockConnected
 
   let total = file.getFileSize()
-  var data = newString(4000)
+  var data = alloc(4000)
```

Comment:
Why `alloc` here? `newString` makes more sense and is less dangerous.

Reply:
I changed it back to use string